src/dashboard/components/DonationSummary.vue

```diff
@@ -121,6 +129,27 @@ export default {
     }),
   },
   methods: {
+    changeDonationAmount(creator, new_value) {
+      const dist =
+        Object.keys(this.$store.state.metamask.distribution).length > 0
+          ? this.$store.state.metamask.distribution
+          : this.creators.reduce((obj, c) => {
+              obj[c.id] = c.share;
+              return obj;
+            }, {});
+      const redist_targets = _.omit(dist, [creator.id]);
+      const unchanged_share_sum = _.sum(Object.values(redist_targets));
+      const change = new_value / 100 - dist[creator.id];
+      const new_dist = _.mapValues(redist_targets, share =>
+        unchanged_share_sum > 10e-3
```

Comment:
Could you replace this ternary with an if-else? I find this a bit difficult to read.